packages/client/src/orders/getGuestOrder.ts

```diff
@@ -1,21 +1,21 @@
 import { adaptError } from '../helpers/client/formatError';
 import client from '../helpers/client';
-import type { GetGuestOrders } from './types';
+import type { GetGuestOrder } from './types';
 
 /**
- * Method responsible for fetching the details of a guest user order list.
+ * Method responsible for fetching the details of a guest user order.
  *
  * @param config - Custom configurations to send to the client instance (axios).
  *
  * @returns Promise that will resolve when the call to the endpoint finishes.
  */
-const getGuestOrders: GetGuestOrders = config => {
+const getGuestOrder: GetGuestOrder = (orderId, config) => {
```

Comment:
This new client is added but no matching action is added in redux. I know that this client and the `getOrder` client are both handled by the same code by the backend but we either have 2 choices here:

- Remove this endpoint altogether and use only the `getOrder` and `fetchOrder` to handle it.
- Add a matching `fetchGuestOrder` factory that uses this client but dispatches `FETCH_ORDER_` actions (just like `fetchGuestOrderLegacy` does currently).

I am leaning towards removing this client altogether to avoid confusion with the consumers but discuss this with the team what is the best approach here.

Reply:
You are right, we decided to remove it.. I just forgot to delete it 😅
Done!